proto/tendermint/abci/types.proto

```diff
@@ -21,21 +21,21 @@ import "gogoproto/gogo.proto";
 
 message Request {
   oneof value {
-    RequestEcho               echo                 = 2;
```

Comment:
was this two because there use to be something for 1?

Reply:
think so, but it was a long time ago